python/snewpy/snowglobes.py

```diff
@@ -335,27 +188,52 @@ def simulate(SNOwGLoBESdir, tarball_path, detector_input="all", verbose=False, *
     if verbose:  # Deprecated since SNEWPY v1.2
         warn(f"The 'verbose' parameter to 'snewpy.snowglobes.simulate()' is deprecated and should not be used.", FutureWarning)
 
-    sng = SimpleRate(base_dir=SNOwGLoBESdir, detectors=detector_input, detector_effects=detector_effects)
-    detector_input = list(sng.detectors)
-
+    rc = RateCalculator(base_dir=SNOwGLoBESdir)#, detectors=detector_input, detector_effects=detector_effects)
+    if detector_input == 'all':
+        detector_input = list(sng.detectors)
```

Comment:
```suggestion
    rc = RateCalculator(base_dir=SNOwGLoBESdir)
    if detector_input == 'all':
        detector_input = list(rc.detectors)
```
One more fix: Rename `sng` to `rc`.
(Also, delete commented-out old code.)